packages/app-admin-ui/client/components/CreateItemModal.js

```diff
@@ -222,6 +222,7 @@ const CreateItemModal = ({ prefillData = {}, onClose, onCreate }) => {
                           warnings={validationWarnings[field.path] || []}
                           onChange={onChange}
                           renderContext="dialog"
+                          isReadOnly={field.isReadOnly}
```

Comment:
@MadeByMike this disables the field on Create dialog. this will break anyone who has it required and also sets them as read only.

Reply:
Yeah... that makes sense. People can just not do that... or initialise with a hook.

Reply:
I'm wondering if it would make more sense to completely remove any field that has `isReadOnly` set? Is there any reason to show an empty, disabled form element? In the update case I can see it being useful in order to see the value, but I'm not sure I see any value in the `create` case?